README.md

```diff
@@ -1,93 +1,55 @@
 [//]: # "This README.md file is auto-generated, all changes to this file will be lost."
-[//]: # "To regenerate it, use `npm run generate-scaffolding`."
+[//]: # "To regenerate it, use `python -m synthtool`."
 <img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
 
 # [Google Cloud Storage: Node.js Client](https://github.com/googleapis/nodejs-storage)
 
-[![release level](https://img.shields.io/badge/release%20level-general%20availability%20%28GA%29-brightgreen.svg?style&#x3D;flat)](https://cloud.google.com/terms/launch-stages)
+[![release level](https://img.shields.io/badge/release%20level-general%20availability%20%28GA%29-brightgreen.svg?style=flat)](https://cloud.google.com/terms/launch-stages)
 [![npm version](https://img.shields.io/npm/v/@google-cloud/storage.svg)](https://www.npmjs.org/package/@google-cloud/storage)
 [![codecov](https://img.shields.io/codecov/c/github/googleapis/nodejs-storage/master.svg?style=flat)](https://codecov.io/gh/googleapis/nodejs-storage)
 
-> Node.js idiomatic client for [Cloud Storage][product-docs].
+Cloud Storage Client Library for Node.js
 
-[Cloud Storage](https://cloud.google.com/storage/docs) allows world-wide storage and retrieval of any amount of data at any time. You can use Google Cloud Storage for a range of scenarios including serving website content, storing data for archival and disaster recovery, or distributing large data objects to users via direct download.
```

Comment:
Yeah, it would be really nice to hang onto this, I think.  Not sure if putting the long description in the metadata or embedding it in a `README.md.tpl` file is the right path.

Reply:
I propose the late-game addition of `readme-partials.yml`, with a section `introduction`. This would give us the flexibility to gradually hand-write some portions of the API documentation, which I think we're going to inevitably want for our libraries.

Reply:
In this case it looks like it's copied from https://cloud.google.com/storage/docs/ verbatim.

Reply:
@bcoe I really think this is a *great* idea.  We had a few issues come up in the past asking for this, even with nodejs-repo-tools.